### ISSUE TITLE
fcn-siftflow stops training

### ISSUE THREAD

**Issue by Myzhar** [NVIDIA/DIGITS#1879]
Hi all,

I tried to train an FCN-SIFTFLOW (fcn82 and fcn32s) segmentation network, but DIGITS servers seems to freeze without any error (https://github.com/shelhamer/fcn.berkeleyvision.org)

Any ideas?

**Comment by Hemofektik** [comment]
Do you use a pretrained transplanted model as a basis for training as it is described in the [Segmentation Example](https://github.com/NVIDIA/DIGITS/tree/master/examples/semantic-segmentation)?

**Comment by Myzhar** [comment]
Yes, I trained a lot of SegNets using other pretrained models. I left it running for a lot and after hours it starts to output a few of results, but it is really slow and DIGITS server (browser frontend) does not works.
I think that FCN-SIFTFLOW is really too heavy for my dataset
